lib/acts_as_list/active_record/acts/list.rb

```diff
@@ -458,7 +458,7 @@ def check_scope
           end
 
           def reload_position
-            self.reload
+            self.reload unless new_record?
```

Comment:
It is not required. I left it there because I wanted to change as little as possible.